src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -231,24 +212,22 @@ const TableHeader: React.FC<TableHeaderProps> = ({
         <div className="table-header__lock-indication">
           {isCurrentTableLocked ? "Table Locked" : "Lock Table"}
         </div>
-        <label className="switch table-header__lock-toggle">
-          <input
-            type="checkbox"
-            className="switch-hidden-input"
-            checked={!!isCurrentTableLocked}
-            onChange={toggleIsCurrentTableLocked}
-          />
-          <span className="slider" />
-        </label>
+        <Toggler
+          toggled={!!isCurrentTableLocked}
+          containerClassName="table-header__lock-toggle"
+          onChange={toggleIsCurrentTableLocked}
+        />
       </div>
 
       <EditTableTitleModal
         isShown={isShown}
         title={tableTitle}
-        error={error}
+        description={tableSubtitle}
```

Comment:
Why do we call it subtitle here, and description elsewhere? They should be consistent.

Reply:
From memory of the discussions around this feature, there was a distinction between a subtitle and the description as well. I don't believe they were the same thing.

Reply:
https://github.com/sparkletown/sparkle/pull/1377#discussion_r632474494

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/4b7532e6f02068126c212a6f0dab1714aaaad55a